CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 11.0.0
+  - Update kafka client to 3.3.1, requires Logstash >= 8.3.0. Removed `default` value for setting `client_dns_lookup`, this value can't be used.
```

Comment:
Do we know what the previous value of `default` mapped to?

Reply:
In Kafka `client.dns.lookup` is used to define the policy to select one if IPs exposed in a A record in the DNS.
`default` means "take the first". The problem with this, is that if a cluster is composed on multiple nodes, and the first node listed in the A record has problems, then all the client (consumer and producer) application experience an outage despite the cluster is healthy because all other nodes are alive.
This value was deprecated in Kafka `2.3` and removed in Kafka `3.0`.
Now the default value is `use_all_dns_ips` which means use the first resolved IP which has a connection success.